psalm.xml

```diff
@@ -37,6 +37,7 @@
 				<referencedClass name="Automattic\WooCommerce\StoreApi\Utilities\JsonWebToken" />
 				<referencedClass name="Automattic\WooCommerce\StoreApi\SessionHandler" />
 				<referencedClass name="Automattic\WooCommerce\Utilities\PluginUtil" />
+				<referencedClass name="Automattic\WooCommerce\Utilities\FeaturesUtil" />
```

Comment:
I had to suppress the error since `ERROR: UndefinedClass` is still triggered, even with a `class_exists()` check in place.